src/legacy/core_plugins/tile_map/public/tile_map_fn.js

```diff
@@ -43,6 +43,10 @@ export const createTileMapFn = () => ({
       geocentroid,
     });
 
+    if (geohash && geohash.accessor) {
+      convertedData.meta.geohash = context.columns[geohash.accessor].meta;
```

Comment:
keep the geohash column meta information, so it can be used to construct filters